lib/tasks/dummy_declarations.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# frozen_string_input: true
+#
+class DummyDeclarations
+  class << self
+    def call
+      LeadProvider.all.each do |lp|
+        cpd = lp.cpd_lead_provider
+        lp.ecf_participants.each do |p|
```

Comment:
Full words in variable names please.

Reply:
And maybe we want to add a check here to make sure this can't be executed in sandbox or production.